evm_loader/program/src/instruction/storage_to_v2/mod.rs

```diff
@@ -0,0 +1,10 @@
+use solana_program::pubkey::Pubkey;
+
+pub mod write_value_to_distributed_storage;
+pub mod convert_data_account_from_v1_to_v2;
+
+// Migration operator public key: 6sXBjtBYNbUCKFq3CuAg7LHw9DJCvXujRUEFgK9TuzKx
+const OPERATOR_PUBKEY: Pubkey = Pubkey::new_from_array([
```

Comment:
You can use `const OPERATOR_PUBKEY: Pubkey = pubkey!("6sXBjtBYNbUCKFq3CuAg7LHw9DJCvXujRUEFgK9TuzKx");

Reply:
Thanks, fixed.